From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 17 Aug 2026 06:39:28 +0000
Subject: Translated using Weblate (Turkish)

Currently translated at 100.0% (5 of 5 strings)

Translation: openkylin-huanghe/stylus-settings
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/stylus-settings/tr/
---
 src/stylus-settings/translations/tr.ts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/stylus-settings/translations/tr.ts b/src/stylus-settings/translations/tr.ts
index 530abdf..3618e79 100644
--- a/src/stylus-settings/translations/tr.ts
+++ b/src/stylus-settings/translations/tr.ts
@@ -25,17 +25,17 @@
     <message>
         <location filename="../stylussettingpage.ui" line="67"/>
         <source>Stylus Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Stilus Ayarları</translation>
     </message>
     <message>
         <location filename="../stylussettingpage.ui" line="162"/>
         <source>Please further configure the side button function of the stylus through other software</source>
-        <translation type="unfinished"></translation>
+        <translation>Lütfen stilusun kenar düğmesi işlevini başka bir yazılım aracılığıyla daha fazla yapılandırın</translation>
     </message>
     <message>
         <location filename="../stylussettingpage.ui" line="234"/>
         <source>Use the side button 2 of the stylus to simulate Erase</source>
-        <translation type="unfinished"></translation>
+        <translation>Silgi simülasyonu için stilusun kenar düğmesi 2&apos;yi kullan</translation>
     </message>
 </context>
 <context>
@@ -43,7 +43,7 @@
     <message>
         <location filename="../stylussettingpage.cpp" line="48"/>
         <source>Stylus Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Stilus Ayarları</translation>
         <extra-contents_path>/Stylus/Stylus Settings</extra-contents_path>
     </message>
 </context>
@@ -56,7 +56,7 @@
     <message>
         <location filename="../stylus-settings.cpp" line="42"/>
         <source>Stylus</source>
-        <translation type="unfinished"></translation>
+        <translation>Stilus</translation>
     </message>
 </context>
 <context>
